11ty/definitions/mansplaining.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Mansplaining
+slug: mansplaining
+speech: verb
+defined: true
+reading:
+- text: "Rebecca Solnit's 'Men Explain Things to Me' (2008)"
+  href: https://www.guernicamag.com/rebecca-solnit-men-explain-things-to-me/
+- text: "Kim Goodwin's 'Am I mansplaining? graphic"
+  href: https://ichef.bbci.co.uk/wwfeatures/live/1600_900/images/live/p0/6f/zj/p06fzjxg.jpg
+---
+
+A man explaining a topic to a woman in a condescending, overconfident way, without acknowledging the fact that the woman often knows more about that topic than the man doing the explaining. 
```

Comment:
```suggestion
when someone, often a man, explains a topic to a woman in a condescending, overconfident way, without acknowledging the fact that the person being explained to, often a woman, often knows more about that topic than the explainer. 
```

Reply:
While I think it's important that it's usually men doing this to women, I do think that we need to show that mansplaining anyone can do if they have relative or systemic power over somebody else.

Reply:
I would also suggest adding an "Issues" and "Impact" section, as they're somewhat core to the dictionary. Here is a rough first-pass:

```suggestion
## Issues
Mansplaining places men (usually without authority) in a position of default authority, and places women and gender minoritised people with authority in a position of default subordination, which reinforces the power differential in men's favour.

Mansplaining occupies real time in meetings, leading to men gaining more air time and exposure, despite the stereotype that women speak more.[1](https://slate.com/human-interest/2013/02/do-women-talk-more-the-answer-is-no-but-the-belief-persists-despite-the-evidence.html)[2](https://slate.com/human-interest/2013/02/do-women-talk-more-the-answer-is-no-but-the-belief-persists-despite-the-evidence.html)

## Impact
Mansplaining can reinforce toxic masculinity and rape culture, as it fundamentally is about disbelieving women and gender-minoritised individuals.

Mansplaining contributes to the invalidation and disbelief of any women and gender-minoritised individuals and their credentials. The disbelief and intrinsic questioning of women and gender-minoritised individual especially reinforces systemic bias and injustice against them.

Someone who is repeatedly mansplained can be perceived as lacking credentials or knowledge they very well may have, which can lead to missed career opportunities or systematic exclusion from leadership, for example.
```